import error_check as er
er.check()
